src/main/java/mate/academy/spring/boot/controller/BookController.java

```diff
@@ -17,39 +21,46 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book API")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books")
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
     @GetMapping("/{id}")
+    @Hidden
```

Comment:
Why hidden?

Reply:
I used this annotation because I don’t think I should share methods of removal by ID and search by ID, it’s not an article)